assets/js/editor/Editor.tsx

```diff
@@ -127,10 +160,12 @@ async function loadDTS(specifier: string): Promise<Lib[]> {
       if (!filePath.startsWith('node_modules')) {
         // Load every common typedef into the common module
         let content = await fetchFile(`${commonSpecifier}${filePath}`);
-        content = content.replace(/\* +@(.+?)\*\//gs, '*/');
-        results.push({
-          content: `declare module '@openfn/language-common' { ${content} }`,
-        });
+        if (!content.match(/<!doctype html>/i)) {
```

Comment:
TODO check this doctype stuff - this is from `npm serve` returning folders, I think. When the plug is enabled I'll come back and have another look.